axiom/store.py

```diff
@@ -1373,6 +1375,9 @@ def _loadExistingIndexes(self):
     def _initdb(self, dbfname):
         self.connection = Connection.fromDatabaseName(dbfname)
         self.cursor = self.connection.cursor()
+        if self.journalMode is not None:
+            self.querySchemaSQL(
+                'PRAGMA *DATABASE*.journal_mode = {}'.format(self.journalMode))
```

Comment:
Any reason you're doing your own string interpolation instead of using `?` and passing args to `querySchemaSQL`?

Reply:
`?` doesn't work in this position, sadly.